src/config.rs

```diff
@@ -297,10 +297,11 @@ pub enum Config {
     #[strum(props(default = "0"))]
     DownloadLimit,
 
-    /// Send sync messages, requires `BccSelf` to be set as well.
-    /// In a future versions, this switch may be removed.
+    /// Enable sending and executing sync messages. Sending requires `BccSelf` as well.
+    /// Earlier executing was always enabled, the messages are force-encrypted anyway. But for users
+    /// it's probably more clear whether a device is synchronised or not.
```

Comment:
- What does `executing` mean here? Is sending not the execution?
- `Sending requeres BccSelf **as well**` leaves open the question, what the other thing it needs is? Maybe state it or remove the `as well` I would suggest.
- The two following sentences are talk about old code and legitimation for this. I don't think we need this.

This is a lot of nitpicking, so if you don't agree with me here, you can you rust resolve this without changing a thing.